vmdk_plugin/utils/fs/device_watcher_windows.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Device watching logic for Windows OS.
+
+package fs
+
+import (
+	"fmt"
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	devChangeTimeout = 1 // Number of seconds to wait for a Win32_DeviceChangeEvent
+
+	// PowerShell script to register and wait for a Win32_DeviceChangeEvent.
```

Comment:
please add a comment with something like "we go with PowerShell because there is no functional WMI  GO library"